blocks/hero/hero.css

```diff
@@ -154,4 +192,18 @@ main .hero > div:nth-of-type(n+2) div:nth-of-type(2) p:nth-of-type(n+2) > pictur
     display: block;
     padding-left: 50px;
   }
+
+  main .hero.home::after {
+    background: url('/icons/bg_hero-home-section-2-arrow.png');
+    background-size: 44px 20px, cover;
+  }
+
+  main .hero.home .btn-scroll img {
+    width: 32px;
+    height: 32px;
+  }
+
+  main .hero.home .btn-scroll a {
+    font-size: 1.6rem;
```

Comment:
would it make sense to use --body-font-size-m here or is this a special case?

Reply:
You mean `--heading-font-size-s` right?

Reply:
good point!